include/aos/iam/modules/config.hpp

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
Missing discussion:

No need to move cert modules options into separate file. Except ones, directly related to pkcs11 module. Also, we should have possibility to enable/disable this module compile time. So, I think all pkcs11 module files should be move into separate pkcs11 module folder.
Author
@[mykola-kobets-epam](https://github.com/mykola-kobets-epam) mykola-kobets-epam [Dec 8, 2023](https://github.com/aoscloud/aos_core_lib_cpp/pull/47#discussion_r1420296255)

    No need to move cert module options into separate file
    In this case it's not clear where all of this options should be located in. Also I want to minimize dependencies from the upper level folders within a module (as its hard to track interrelationships within a module when you have header files included from upper and lower dir level). Currently I'm trying to add all necessary data closer to the context of use.

    Also, we should have possibility to enable/disable this module compile time.
    I think it would better to implement this option when all of the related parts are implemented. Probably I'll add it in cryproki wrapper PR.

    pkcs11 module files should be move into separate pkcs11 module folder
    Added pkcs11 folder, moved some config options there.